Makefile

```diff
@@ -1,10 +0,0 @@
-
```

Comment:
Okay for this PR I am not modifying anything here, I'll restore back the Makefile. But it is to be noted that `test-no-multiprocessing` remains unused.